test-suites/_base_interest_rate_strategy.spec.ts

```diff
@@ -238,6 +244,30 @@ describe("Interest Rate Tests", () => {
         )
       ).to.be.revertedWith(INVALID_OPTIMAL_USAGE_RATIO);
     });
+
+    it("TC-interest-rate-strategy-07 PoolConfigurator updates the ReserveInterestRateStrategy address", async () => {
+      const {pool, deployer, dai, configurator} = await loadFixture(
+        testEnvFixture
+      );
+
+      // Impersonate PoolConfigurator
+      await topUpNonPayableWithEther(
+        deployer.signer,
+        [configurator.address],
+        utils.parseEther("1")
+      );
+      await impersonateAccountsHardhat([configurator.address]);
+      const configSigner = await hre.ethers.getSigner(configurator.address);
+
+      expect(
+        await pool
+          .connect(configSigner)
+          .setReserveInterestRateStrategyAddress(dai.address, ZERO_ADDRESS)
+      );
```

Comment:
```suggestion
      expect(
        await pool
          .connect(configSigner)
          .setReserveInterestRateStrategyAddress(dai.address, ZERO_ADDRESS)
      );
```
`expect(x);`  inside It seems that no verification has been done